tailwind.config.js

```diff
@@ -12,9 +12,13 @@ module.exports = {
     spacing: {
       quarter: '5px',
       half: '10px',
+      threeQuarter: '15px',
       base: '20px',
       double: '40px',
     },
+    borderRadius: {
+      default: '6px',
+    },
```

Comment:
Are shadows included in this?

Reply:
Ah I missed this config, will add now since our inline boys use a shadow